test/voice_registration.test.js

```diff
@@ -1505,6 +1547,37 @@ describe("Mama Nigeria App", function() {
                         .run();
                 });
             });
+
+            describe("if it is *", function() {
+                it("should repeat", function() {
+                    return tester
+                        .setup.user.addr('07030010001')
+                        .inputs(
+                            {session_event: 'new'}
+                            , '12345'       // state_personnel_auth
+                            , '6'           // state_msg_receiver - friend_only
+                            , '09092222222' // state_msisdn
+                            , '2'           // state_pregnancy_status - baby
+                            , '2'           // state_pregnancy_status - babyk
+                            , '2'           // state_baby_birth_year - last year
+                            , '11'          // state_baby_birth_month - nov
+                            , '*'           // state_baby_birth_day
+                        )
+                        .check.interaction({
+                            state: 'state_baby_birth_day',
+                            reply: 'Birth day in 11 2016'
+                        })
+                        .check.reply.properties({
+                            helper_metadata: {
+                                voice: {
+                                    speech_url: 'http://localhost:8004/api/v1/eng_NG/state_baby_birth_day_11.mp3',
+                                    wait_for: '#'
+                                }
+                            }
+                        })
+                        .run();
+                });
+            });
```

Comment:
We should test what happens to states with creator_opts being used to pass through information.